Source/Csla/Core/BusinessBase.cs

```diff
@@ -3017,10 +3017,10 @@ protected void LoadPropertyAsync<R>(PropertyInfo<R> property, Task<R> factory)
     #endregion
 
     #region IsBusy / IsIdle
-
-    [NonSerialized]
-    [NotUndoable]
-    private bool _isBusy;
+    /// <summary>
+    /// Indicates object is busy or not 
+    /// </summary>
+    private int _isBusy;
```

Comment:
Sorry my bad here. Please name it `_isBusyCounter` and add the attributes back.

Reply:
Done